cluster/pv-repo.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: config.porch.kpt.dev/v1alpha1
+kind: PackageVariant
+metadata:
+  name: example-repo
+spec:
+  upstream:
+    package: nephio-repo
+    repo: blueprint-manifests
```

Comment:
I think repo and revision have to be changed here

Reply:
repo: nephio-example-packages
    revision: v1